src/core/bittorrent/torrentinfo.h

```diff
@@ -75,8 +75,11 @@ namespace BitTorrent
         QList<TrackerEntry> trackers() const;
         QList<QUrl> urlSeeds() const;
         QByteArray metadata() const;
+        libtorrent::file_storage files() const;
+        libtorrent::file_storage origFiles() const;
```

Comment:
Drop these methods (and remapFiles()) from TorrentInfo interface, because it contradicts the new code design. If the Session needs to use any native features, it should do so through nativeInfo().